<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="chouman" failures="3" tests="4" name="PetStoreTest" time="19.925" errors="0" timestamp="2023-04-01T12:41:29 EDT">
  <testcase classname="testCases.TC_002_LoginTest" name="test_Login" time="3.395"/>
  <testcase classname="testCases.TC_002_LoginTest" name="test_loginDDt" time="2.083">
    <failure type="java.lang.AssertionError" message="test failed">
      <![CDATA[java.lang.AssertionError: test failed
at testCases.TC_002_LoginTest.test_loginDDt(TC_002_LoginTest.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- test_loginDDt -->
  <testcase classname="testCases.TC_002_LoginTest" name="test_loginDDt" time="2.698">
    <failure type="java.lang.AssertionError" message="test failed">
      <![CDATA[java.lang.AssertionError: test failed
at testCases.TC_002_LoginTest.test_loginDDt(TC_002_LoginTest.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- test_loginDDt -->
  <testcase classname="testCases.TC_002_LoginTest" name="test_loginDDt" time="2.687">
    <failure type="java.lang.AssertionError" message="test failed">
      <![CDATA[java.lang.AssertionError: test failed
at testCases.TC_002_LoginTest.test_loginDDt(TC_002_LoginTest.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- test_loginDDt -->
</testsuite> <!-- PetStoreTest -->
